corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -5,7 +5,7 @@
   <div tabindex="-1" class ="gr" data-bind="
         css: {
           'gr-no-children': $data.children().length === 0,
-          'gr-has-no-questions': _.all($data.children(), function(d) { return d.type() !== 'grouped-question-tile-row' }),
+          'gr-has-no-questions': $data.children().every(function(d) { return d.type() !== 'grouped-question-tile-row' }),
```

Comment:
Not that it matters too much, but if there is an `any` method doing `!$data.children().any(function(d) { return d.type() === 'grouped-question-tile-row'` could be faster since it can return as soon as it finds once such child and does not have to check all of them.

Reply:
`every` should also short circuit when it encounters a false value